fsm/leave_sync_nses.go

```diff
@@ -68,7 +68,7 @@ func LeaveSyncNsesAction(z *music.Zone) bool {
 		return false
 	}
 
-	nsrem := []dns.RR{}
+	var nsrem []dns.RR
```

Comment:
Ok, so you don't like to instantiate a new variable with an assignment. Does this mean (I'm really curious) that you're against constructs like:

foo := map[string]*dns.RR{}

and would rather have:

var foo map[string]*dns.RR

?

Reply:
My understanding that is better to declare an empty slice with:
` var myslice  []stuff` 
than with : 
`myslice :=  []stuff{}`
according to this: https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices

I don't think this applies to Maps.

So this isn't a personal opinion as much as me trying to write better go code.

Reply:
Why is it better? Both are legal constructs. And using assignment means we use the same construct for both slices and maps, which makes the code more uniform. That said, my objection is not about the construct

var foo []stuff

but spending time on changing one perfectly legal contruct for another perfectly legal construct.